sdk-lib/src/androidTest/java/cash/z/ecc/android/sdk/test/Global.kt

```diff
@@ -3,9 +3,25 @@ package cash.z.ecc.android.sdk.test
 import android.content.Context
 import androidx.annotation.StringRes
 import androidx.test.core.app.ApplicationProvider
+import kotlinx.coroutines.channels.awaitClose
+import kotlinx.coroutines.flow.callbackFlow
+import kotlin.test.assertNotNull
 
 fun getAppContext(): Context = ApplicationProvider.getApplicationContext()
 
 fun getStringResource(@StringRes resId: Int) = getAppContext().getString(resId)
 
 fun getStringResourceWithArgs(@StringRes resId: Int, vararg formatArgs: String) = getAppContext().getString(resId, *formatArgs)
+
+fun readFileLinesInFlow(filePathName: String) = callbackFlow {
+    val testFile = javaClass.getResourceAsStream(filePathName)
+    assertNotNull(testFile, "Test file read failure.")
+
+    val reader = testFile.bufferedReader()
+    reader.forEachLine { line ->
+        trySend(line)
+    }.also {
+        close()
```

Comment:
I think a `bufferedReader.use` would simplify this and eliminate the `close()`

Reply:
We'd use a function `use` here to close the buffer after all lines are read, fortunately `forEachLine` closes it for us at the end. I assumed that the closed buffer would then close the flow, but it didn't happen, so that's why I used `also { close () }` to close the flow after read. Do you see any better solution here, which wouldn't require change on the caller side?